packages/css/src/input.css

```diff
@@ -7,7 +7,7 @@
   --dsc-input-background--readonly: var(--ds-color-neutral-background-subtle);
   --dsc-input-background: var(--ds-color-neutral-background-default);
   --dsc-input-border-color--checked: var(--ds-color-base-default);
-  --dsc-input-border-color--invalid: var(--ds-color-danger-border-strong);
+  --dsc-input-border-color--invalid: var(--ds-color-danger-text-subtle);
```

Comment:
We need an explanation as to why we use a **text token** for the **border** color here

Reply:
Changed css variable and added description as to how its used. 

We also used `text-subtle` as its guaranteed to have enough contrast against background and surface, as is not the case for `border-strong`.

Reply:
Refactored variables now for simpler usage and linking to props according to state.